spec/views/spree/layouts/admin/_login_nav.html.haml_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'spree/layouts/admin/_login_nav.html.haml' do
+
+  it 'display the application version in admin navbar' do
+    def  view.spree_current_user
+      FactoryGirl.build(:user)
+    end
```

Comment:
Thanks for explaning, that makes sense (even though it's ugly). I guess this is enough of a common practice not to warrant an explanation in the comment.